src/js/gi/components/profile/AdditionalInformation.jsx

```diff
@@ -30,9 +32,48 @@ export class AdditionalInformation extends React.Component {
           &nbsp;<a href={it.vetWebsiteLink} target="_blank">
             View policy
           </a>
-        </p>
+        </div>
       );
 
+    return (
+      <div className="institution-summary">
+        <h3>Institution summary</h3>
+        <div>
+          <strong>
+            <a onClick={this.props.onShowModal.bind(this, 'accredited')}>
+              Accredited:
+            </a>
+          </strong>
+          &nbsp;{it.accredited ?
+            <span>Yes (<a href={`http://nces.ed.gov/collegenavigator/?id=${it.cross}#accred`} target="_blank">
+              See accreditors
+            </a>)</span> : 'No'}
+        </div>
+        {typeOfAccreditation}
+        {vetTuitionPolicy}
+        <div>
+          <strong>
+            <a onClick={this.props.onShowModal.bind(this, 'singleContact')}>
+              Single point of contact for veterans:
+            </a>
+          </strong>
+          &nbsp;{!!it.vetPoc ? 'Yes' : 'No'}
+        </div>
+        <div>
+          <strong>
+            <a onClick={this.props.onShowModal.bind(this, 'creditTraining')}>
+              Credit for military training:
+            </a>
+          </strong>
+          &nbsp;{!!it.creditForMilTraining ? 'Yes' : 'No'}
+        </div>
+      </div>
+    );
+  }
```

Comment:
Just moved the deleted block below into its own function so it's clearer to see the condition in which it's rendered.